### PR TITLE
Release JxBrowser 8.0.0

### PR DESCRIPTION
**Pull request by vladimir-ikryanov**
Update JxBrowser version to 8.0.0 and add Gradle Quick Start for Compose.

### CODE REVIEW

File: Gradle/Compose/README.md [hunk 1]
```diff
@@ -1,28 +1,28 @@
-# JxBrowser in Compose Gradle Project
+# JxBrowser in Swing Gradle Project
```
Comment (vlad-lubenskyi) [inline Gradle/Compose/README.md]: This file seems to have been changed by mistake.

File: Gradle/Compose/build.gradle.kts [hunk 2]
```diff
@@ -26,7 +26,7 @@ plugins {
     id("com.teamdev.jxbrowser") version "1.1.0"
 
     // Adds UI toolkits.
-    id("org.jetbrains.compose") version "1.6.11"
+    id("org.jetbrains.compose") version "1.7.0-rc01"
```
Comment (vlad-lubenskyi) [inline Gradle/Compose/build.gradle.kts:29]: Do we rely on RC?

File: Gradle/Compose/src/main/kotlin/HelloCompose.kt [hunk 3]
```diff
@@ -34,14 +34,14 @@ import com.teamdev.jxbrowser.view.compose.BrowserView
...
     val browser = remember { engine.newBrowser() }
 
     BrowserView(browser)
 
     LaunchedEffect(Unit) {
-        browser.navigation.loadUrl("https://html5test.teamdev.com")
+        browser.navigation.loadUrl("https://ui.shadcn.com/blocks")
     }
-}
+}
```
Comment (vlad-lubenskyi) [inline Gradle/Compose/src/main/kotlin/HelloCompose.kt:47]: Please add the empty line.

File: Gradle/Compose/src/main/kotlin/HelloCompose.kt [hunk 4]
```diff
@@ -34,14 +34,14 @@ import com.teamdev.jxbrowser.view.compose.BrowserView
...
+    state = WindowState(width = 700.dp, height = 500.dp),
 ) {
     val engine = remember { Engine(OFF_SCREEN) }
     val browser = remember { engine.newBrowser() }
 
     BrowserView(browser)
 
     LaunchedEffect(Unit) {
-        browser.navigation.loadUrl("https://html5test.teamdev.com")
+        browser.navigation.loadUrl("https://ui.shadcn.com/blocks")
```
Comment (vlad-lubenskyi) [inline Gradle/Compose/src/main/kotlin/HelloCompose.kt:45]: Is this intentional?

**Review by vlad-lubenskyi — CHANGES_REQUESTED** [review]

File: Gradle/Compose/build.gradle.kts (hunk 2 above)
Comment (vladimir-ikryanov) [reply Gradle/Compose/build.gradle.kts:29]: Yes.

File: Gradle/Compose/src/main/kotlin/HelloCompose.kt (hunk 4 above)
Comment (vladimir-ikryanov) [reply Gradle/Compose/src/main/kotlin/HelloCompose.kt:45]: Fixed.

**Review by Danil-Didkovskiy — APPROVED** [review]

**Pull request merged**